modules/react-native-status/android/src/main/java/im/status/ethereum/module/DatabaseManager.kt

```diff
@@ -23,16 +25,35 @@ class DatabaseManager(private val reactContext: ReactApplicationContext) : React
 
     @ReactMethod
     fun exportUnencryptedDatabase(accountData: String, password: String, callback: Callback) {
-        Log.d(TAG, "login")
+        Log.d(TAG, "exportUnencryptedDatabase")
 
         val newFile = getExportDBFile()
 
         utils.migrateKeyStoreDir(accountData, password)
-        val result = Statusgo.exportUnencryptedDatabase(accountData, password, newFile.absolutePath)
-        if (result.startsWith("{\"error\":\"\"")) {
-            Log.d(TAG, "Login result: $result")
-        } else {
-            Log.e(TAG, "Login failed: $result")
+        
+        try {
+            val accountJson = JSONObject(accountData)
+            
+            val params = JSONObject().apply {
+                put("account", accountJson)
+                put("password", password)
+                put("databasePath", newFile.absolutePath)
+            }
+            
+            val jsonParams = params.toString()
+            
+            val result = Statusgo.exportUnencryptedDatabaseV2(jsonParams)
+            if (result.startsWith("{\"error\":\"\"")) {
+                Log.d(TAG, "Export result: $result")
+            } else {
+                Log.e(TAG, "Export failed: $result")
+            }
+
+            callback.invoke(newFile.absolutePath)
```

Comment:
Quick question: If the DB export fails, should we still need to invoke the callback?